libraries/heft-config-file/src/ConfigurationFile.ts

```diff
@@ -98,7 +107,24 @@ export interface IJsonPathMetadata {
    * If this property describes a filesystem path, use this property to describe
    * how the path should be resolved.
    */
-  pathResolutionMethod?: PathResolutionMethod;
+  pathResolutionMethod?: PathResolutionMethod.custom;
+}
+
+/**
+ * Used to specify how node(s) in a JSON object should be processed after being loaded.
+ *
+ * @beta
+ */
+export interface INonCustomJsonPathMetadata {
```

Comment:
I get why this isn't the original name, but feels oddly verbose since "non-custom" implies "normal" or "default". Though I'm not sure what I'd rather call it...

Reply:
Yeah that's where I ended up too. It's marked as `@beta` and it's unlikely someone will actually use this type by name, so we can change it if we come up with something better.